templates/workspace_services/ohdsi/terraform/deploy.sh

```diff
@@ -0,0 +1,10 @@
+export TF_LOG=""
```

Comment:
Does this work? This is called from `make terraform-deploy DIR=./templates/workspace_services/ohdsi`